_posts/2017-08-27-nunit-38.md

```diff
@@ -10,4 +10,6 @@ An issue that caused unit tests to run slower was addressed as was a bug that pr
 
 The Order attribute can now also be applied to the class level to set the order that test fixtures will be run.
 
-You may download NUnit 3.8 from [Github](https://github.com/nunit/nunit/releases). See the [release notes](https://github.com/nunit/docs/wiki/Release-Notes) for more information.
+The 3.8.1 release fixes two critical regressions in the 3.8 release. The first caused the console runner to crash if you are using multiple test parameters. The second issue caused collection constraints checking for multiple items in a collection to fail.
```

Comment:
The crash happens any time the TestParametersDictionary setting is present, right? So I think this happens even without multiple parameters?